<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="WSTIN0130" name="com.disuite.test.thoughtspot.AmazonS3DfCreationTest" tests="2" failures="0" timestamp="12 Nov 2020 08:16:35 GMT" time="241.250" errors="1">
  <testcase name="as3DfCreationTest" time="99.979" classname="com.disuite.test.thoughtspot.AmazonS3DfCreationTest">
    <error type="java.lang.IllegalArgumentException" message="Cannot find elements when the XPath is null.">
      <![CDATA[java.lang.IllegalArgumentException: Cannot find elements when the XPath is null.
at org.openqa.selenium.By$ByXPath.<init>(By.java:380)
at org.openqa.selenium.By.xpath(By.java:98)
at com.disuite.pages.thoughtspot.SyncProperties.maxIgnoredRows(SyncProperties.java:283)
at com.disuite.pages.thoughtspot.SyncProperties.tsSyncProperties(SyncProperties.java:102)
at com.disuite.pages.thoughtspot.AmazonS3DfCreation.s3DfCreation(AmazonS3DfCreation.java:47)
at com.disuite.test.thoughtspot.AmazonS3DfCreationTest.as3DfCreationTest(AmazonS3DfCreationTest.java:28)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- as3DfCreationTest -->
  <testcase name="as3DfCreationTest" time="141.271" classname="com.disuite.test.thoughtspot.AmazonS3DfCreationTest"/>
</testsuite> <!-- com.disuite.test.thoughtspot.AmazonS3DfCreationTest -->
